server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GetAssertionsTests.java

```diff
@@ -68,186 +72,220 @@
  */
 public class GetAssertionsTests extends TestClassWithDatabase {
 
-    private static final Logger LOGGER = LogManager.getLogger(GetAssertionsTests.class);
-
-    /**
-     * Container for the mock-up database.
-     */
-    private final static PostgreSQLContainer<?> postgres = createTestContainer();
-
-    /**
-     * Endpoint for getting assertions.
-     */
-    final String getAssertionsEndpoint = "/raire/get-assertions";
-
-    /**
-     * JSON suffix.
-     */
-    final String json = "json";
-
-    /**
-     * CSV suffix.
-     */
-    final String csv = "csv";
-
-    /**
-     * Wiremock server for mocking the raire service.
-     * (Note the default of 8080 clashes with the raire-service default, so this is different.)
-     */
-    final WireMockServer wireMockRaireServer = new WireMockServer(8111);
-
-    /**
-     * Base url - this is set up to use the wiremock server, but could be set here to wherever you have the
-     * raire-service running to test with that directly.
-     */
-    private static String baseUrl;
-
-    /**
-     * Database init.
-     */
-    @BeforeClass
-    public static void beforeAll() {
-        postgres.start();
-        Persistence.setProperties(createHibernateProperties(postgres));
+  private static final Logger LOGGER = LogManager.getLogger(GetAssertionsTests.class);
+
+  /**
+   * Container for the mock-up database.
+   */
+  private final static PostgreSQLContainer<?> postgres = createTestContainer();
+
+  /**
+   * Endpoint for getting assertions.
+   */
+  final String getAssertionsEndpoint = "/raire/get-assertions";
+
+  /**
+   * JSON suffix.
+   */
+  final String json = "json";
+
+  /**
+   * CSV suffix.
+   */
+  final String csv = "csv";
+
```

Comment:
Same comment as on another PR, can the port number be defined in test settings? Otherwise, what if 8111 clashes with something running on someone's computer?

Reply:
Good idea - have refactored the TestClassWithDatabase a bit to make sure it does actually read test. properties (which wasn't being read before) and that those port numbers are in there instead of spread all over the different classes.